tests/unit/lms/services/mailchimp_test.py

```diff
@@ -42,7 +36,7 @@ def test_it(self, mailchimp_transactional, caplog):
             )
         ]
         mailchimp_transactional.Client.assert_called_once_with(sentinel.api_key)
-        mailchimp_transactional.Client.return_value.messages.send_template.assert_called_once_with(
+        mailchimp_client.messages.send_template.assert_called_once_with(
```

Comment:
We already had a `mailchimp_client` fixture below which can avoid having to type `mailchimp_transactional.Client.return_value` all the time but we weren't using it. I've gone through and replaced `mailchimp_transactional.Client.return_value` with `mailchimp_client` everywhere